Sync WPT with upstream (13-06-2025)

servo-wpt-sync opened the pull request:
Automated downstream sync of changes from upstream as of 13-06-2025
[no-wpt-sync]

servo-wpt-sync committed:
Update web-platform-tests to revision b'07ed97f2cf084e3680eaad8eb9f0492c5d4ed6b9'

Signed-off-by: WPT Sync Bot <ghbot+wpt-sync@servo.org>

Loirooriol commented:
Some expectations seem wrong:
https://github.com/servo/servo/actions/runs/15634525485
https://github.com/servo/servo/actions/runs/15634993911

Loirooriol committed:
Fix expectations

This commit modified the tests:
https://github.com/web-platform-tests/wpt/commit/9009950cb411674acbc2df661167eef799a10436

Signed-off-by: Oriol Brufau <obrufau@igalia.com>
